setup.py

```diff
@@ -39,18 +39,30 @@
         "Twisted",
         "appdirs",
         "attrs",
-        "toga",
```

Comment:
Yes. Ish. Depends on the breakage you want :-)

So, the problem is that on mobile platforms, installing `toga` will install the version for the platform you're *building* on, not the platform you're targeting. So - it will install `toga-cocoa` when you're building for iOS, for example.

I've worked around this by adding the `app_requires` flag; but that won't help a `pip install myriagon` installation.

So - it depends what your preferred deployment scheme is. If you're expecting a `pip install ` as the default mechanism to install, then yes, you need to have `toga` in the `install_requires`. But if you're expecting to use Briefcase, it makes more sense to use `app_requires`.

A third option would be to put `toga-core` in `install_requires` - if you do this, you'll get an error the first time you run, telling you that you need to install toga-cocoa or toga-gtk. I suppose this could even be automated - asking the question "do you want to install toga-cocoa", and then actually doing that install.

Any preferences?